src/packages/pickerview/doc.zh-TW.md

```
@@ -0,0 +1,93 @@
+### options 數據結構
+
+| 屬性 | 說明 | 類型 | 默認值 |
+| --- | --- | --- | --- |
+| label | 選項的文字內容 | `string` \| `number` | `-` |
+| value | 選項對應的值，且唯一 | `string` \| `number` | `-` |
+
+## 主題定制
+
+### 樣式變量
+
+組件提供了下列 CSS 變量，可用於自定義樣式，使用方法請參考 [ConfigProvider 組件](#/zh-CN/component/configprovider)。
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)